__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+process.env.NODE_ENV = "test";
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require("./../db/seeds/seed");
+const testData = require("./../db/data/test-data");
+
+afterAll(() => {
+    if (db.end) db.end();
+  });
+  
```

Comment:
Not necessarily an issue, but for my own ocd, before each and after all should be chronological order